publiccode.yml

```diff
@@ -0,0 +1,58 @@
+# This repository adheres to the publiccode.yml standard by including this 
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+name: Foodsoft
+url: 'https://github.com/foodcoops/foodsoft.git'
+landingURL: 'https://foodcoops.net'
+releaseDate: '2025-03-20'
+softwareVersion: v4.9.1
+developmentStatus: stable
+platforms:
+  - web
+softwareType: standalone/web
+categories:
+  - e-commerce
+  - business-process-management
+  - project-collaboration
+  - warehouse-management
+maintenance:
+  type: community
+legal:
+  license: AGPL-3.0-or-later
+localisation:
+  localisationReady: true
+  availableLanguages:
+    - de
+    - en
+    - es
+    - fr
+    - nl
+    - tr
+description:
+  it:
```

Comment:
It's my first encounter with publiccode.yml and it's quite likely that I misunderstand something, but shouldn't this be `en:`, if all the sub values are in English?
Even if not, why only Italian?